examples/init.d/node_exporter

```diff
@@ -4,7 +4,7 @@ RETVAL=0
 PROG="node_exporter"
 EXEC="/etc/node_exporter/node_exporter"
 LOCKFILE="/var/lock/subsys/$PROG"
-OPTIONS="-web.listen-address=:9201"
+OPTIONS="--web.listen-address=:9201"
```

Comment:
Oh, this is the wrong default port number, can you update this to `9100`?

Reply:
Thanks, I've pushed a new commit.